### PR TITLE
177 add filter on caller and callee merge

### PR DESCRIPTION
**Pull request by samer-12**

### CODE REVIEW

File: extractor/main.py [hunk 1]
```diff
@@ -24,15 +24,20 @@ def extractor(self):
         self.header = extractor.getHeader()
         self.body = extractor.getBody()
 
-    def logInterperter(self, sessionID):
+    def logInterperter(self, sessionID, startTime, endTime, sipTo, sipFrom):
+
+        # the time is not being submitted
+        print('main start: ', startTime)
```
Comment (samer-12) [inline extractor/main.py:30]: Keep or delete?

File: sip-viewer/src/app/upload-portal/upload-portal.component.ts [hunk 2]
```diff
@@ -65,6 +67,10 @@ export class UploadPortalComponent {
     if (this.simpleForm.valid) {
       isValid = true;
       this.sessionIDs = this.parseSessionID(this.simpleForm.value.sessionIDs);
+      this.startTime = this.simpleForm.value.startTime;
+      this.endTime = this.simpleForm.value.endTime;
+      console.log(this.startTime);
+      console.log(this.endTime);
```
Comment (samer-12) [inline sip-viewer/src/app/upload-portal/upload-portal.component.ts:73]: Delete?

File: extractor/app.py [hunk 3]
```diff
@@ -16,11 +16,11 @@
 app.config['PROCESSED_FOLDER'] = PROCESSED_FOLDER
 
 
-def processFile(inputFile, logPath, destinationPath, sessionID):
+def processFile(inputFile, logPath, destinationPath, sessionID, startTime, endTime):
     try:
         main_instance = Main(inputFile, logPath, destinationPath)
         main_instance.extractor()
-        main_instance.logInterperter(sessionID)
+        main_instance.logInterperter(sessionID, startTime, endTime)
```
[the remark this answers is not in the document]
Comment (samer-12) [reply extractor/app.py:23]: Fixed elsewhere

File: extractor/main.py (hunk 1 above)
Comment (samer-12) [reply extractor/main.py:30]: Deleting

File: sip-viewer/src/app/upload-portal/upload-portal.component.ts (hunk 2 above)
Comment (samer-12) [reply sip-viewer/src/app/upload-portal/upload-portal.component.ts:73]: Deleting

Comment (hellesvaasand) [reply sip-viewer/src/app/upload-portal/upload-portal.component.ts]: 👍

**Review by hellesvaasand — APPROVED** [review]

**Pull request merged**